packages/core/src/external/carequality/dq/process-inbound-dq.ts

```diff
@@ -56,7 +56,7 @@ export async function processInboundDq(
       log("CCD generated. Fetching the document contents");
     }
 
-    const documentContents = await getDocumentContents(cxId, patientId);
+    const documentContents = await getDocumentContents(cxId, patientId, false);
```

Comment:
This is the only place that calls that function, right?

So why the parameter if we're passing false here?

Also, by sending false we're keeping the current behavior, right?

Reply:
Yes to all, besides the first question. There's another place that calls this function now and it wants a different behavior